web/README.md

```diff
@@ -25,22 +25,34 @@ $ cd oauth2l-web/web
 ### Docker
 Download [Docker](https://docs.docker.com/get-docker/) if not already installed. 

```

Comment:
Shouldn't this also include Docker Compose to be installed?

Reply:
It is already preinstalled with docker.